homeassistant/components/wallbox/coordinator.py

```diff
@@ -78,7 +102,7 @@ def __init__(self, station: str, wallbox: Wallbox, hass: HomeAssistant) -> None:
             update_interval=timedelta(seconds=UPDATE_INTERVAL),
         )
 
-    def _authenticate(self) -> None:
+    def authenticate(self) -> None:
         """Authenticate using Wallbox API."""
         try:
```

Comment:
Is `authenticate` only called from the decorator? If yes, then the try/except block should be removed as it is the same as in the decorator.

Reply:
done, removed try...except